miner/miner_test.go

```diff
@@ -17,6 +18,46 @@ func mustIDAddr(i uint64) address.Address {
 	return a
 }
 
+func TestMessageLimit(t *testing.T) {
+	ctx := context.TODO()
+	a1 := mustIDAddr(1)
+
+	actors := map[address.Address]*types.Actor{
+		a1: {
+			Nonce:   0,
+			Balance: types.NewInt(50000000),
+		},
+	}
+
+	af := func(ctx context.Context, addr address.Address, tsk types.TipSetKey) (*types.Actor, error) {
+		return actors[addr], nil
+	}
+
+	var msgs []types.Message
+
+	for i := 0; i < 550; i++ {
+		t := types.Message{
+			From:     a1,
+			To:       a1,
+			Nonce:    uint64(i),
+			Value:    types.NewInt(1),
+			GasLimit: 50,
```

Comment:
No messages will be selected as 50 gas is below inclusion gas minimum.

Reply:
Are you sure that's being enforced, because it was returning the max limit of messages from the SelectMessages method @Kubuxu ?